engines/toltecs/movie.cpp

```diff
@@ -271,12 +271,16 @@ bool MoviePlayer::handleInput() {
 	Common::EventManager *eventMan = g_system->getEventManager();
 	while (eventMan->pollEvent(event)) {
 		switch (event.type) {
-		case Common::EVENT_KEYDOWN:
-			if (event.kbd.keycode == Common::KEYCODE_ESCAPE)
+		case Common::EVENT_CUSTOM_ENGINE_ACTION_START:
+			switch (event.customType) {
+			case (kActionSkipMovie):
```

Comment:
why?

```suggestion
			case kActionSkipMovie:
```

and it other places